WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsView.swift

```diff
@@ -37,109 +37,37 @@ struct WooShippingCreateLabelsView: View {
     var body: some View {
         NavigationStack {
             ScrollView {
-                VStack(spacing: Layout.verticalSpacing) {
```

Comment:
This part looks like a lot of code change but basically if/else is added to show the content if the store options are loaded and show loading view if not.